### PR TITLE
Move to JuliaData Org

### PR DESCRIPTION
**Pull request by kescobo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [![](http://pkg.julialang.org/badges/YAML_0.7.svg)](http://pkg.julialang.org/?pkg=YAML)
 [![](http://pkg.julialang.org/badges/YAML_1.0.svg)](http://pkg.julialang.org/?pkg=YAML)
 
-[![Build Status](https://travis-ci.org/BioJulia/YAML.jl.svg?branch=master)](https://travis-ci.org/BioJulia/YAML.jl)
+[![Build Status](https://travis-ci.org/JuliaData/YAML.jl.svg?branch=master)](https://travis-ci.org/JuliaData/YAML.jl)
 [![Coverage Status](https://coveralls.io/repos/dcjones/YAML.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/dcjones/YAML.jl?branch=master)
```
Comment (quinnj) [inline README.md:11]: Looks like this coverage status is probably out of date? I prefer codecov myself.

Comment (kescobo) [reply README.md:11]: I'm sure that's true - should I just copy from CSV or something?

Comment (kescobo) [reply README.md:11]: I'll open an issue

Comment (quinnj) [reply README.md:11]: [codecov](https://codecov.io/gh/JuliaData/YAML.jl)

Comment (quinnj) [reply README.md:11]: here's the codecov badge

Comment (quinnj) [reply README.md:11]: can be found here: https://codecov.io/gh/JuliaData/YAML.jl/settings/badge

Comment (kescobo) [reply README.md]: oh neat, I figured I'd have to do some additional configuration, but looks like it's already in the .travis.yaml 👍

**Comment by kescobo** [comment]
This is odd - travis report isn't coming back. When I look in github, it says travis is enabled for JuliaData:

Screen Shot 2020-05-07 at 10 01 07 AM

but on travis' end, it's not:
Screen Shot 2020-05-07 at 10 00 24 AM

**Comment by kescobo** [comment]
I'm gonna go ahead and merge this and sort it out later. Nothing here changed anything that's being tested, so should be fine.

**Pull request merged**